internal/modules/oracle/test.py

```diff
@@ -177,6 +166,23 @@ def test_delegate_feed_consent(self):
         status = query_aggregate_vote(validator1_val["address"])
         self.assertTrue(status)
 
+    # test_high_voting_load spams the nodes with pre-votes and votes,
+    # then verifies voting is avilable next voting period
+    def test_high_voting_load(self):
+        vote_hash = get_hash(EXCHANGE_RATES.ToString(), STATIC_SALT, validator2_val['address'])
+        for i in range(30):
```

Comment:
You want to _really_ spam this like somebody trying to crash the node would:
```suggestion
        for i in range(10000):
```

Reply:
I tested it at a pretty high level to see if it still passed (it did) but I was concerned about run time. I put it at 10,000 and it is still running as I am typing this. I could try out a multi threaded solution, or move it to another test file that doesn't run every PR.

Reply:
@zarazan gotcha. Maybe we could consider making this configurable with an environment variable? but that can be done in a separate PR 

This is good for now 💯